src/scss/partials/_nav.scss

```diff
@@ -54,6 +54,15 @@ $nav-popover-padding          : spacing(x2);
             transition: opacity $nav-transition-duration ease-in-out,
                         z-index 0s linear $nav-transition-duration,
                         left 0s linear ($nav-popover-transition-delay + $nav-popover-transition-duration);
+
+            &.is-visible {
+                overflow-y: scroll;
```

Comment:
Would be good to make this a mixin so that’s not in two different places with the same code (on line 266 as well by the looks of it).

Reply:
Done in #9